randovania/games/fusion/logic_database/Sector 6 NOC.txt

```diff
@@ -709,7 +715,15 @@ Extra - room_id: [18]
   > Pickup (Hidden Power Bomb Tank)
       All of the following:
           Can Use Any Bombs
-          Movement (Beginner) or Can Use Springball
+          Any of the following:
+              All of the following:
+                  # Vanilla Method - freeze a spikey on the crumble blocks (and have regular bombs only)
+                  Any of the following:
+                      # Template not used because normal Ice missiles can't hit the spikey in the right spot
+                      Diffusion Missiles or Can Use Bombs
+                      Ice Beam and Wave Beam
+              # Alternate method - jump onto and out of the crumble blocks with precise timing
+              Movement (Advanced) and Can Use Springball
```

Comment:
is it not possible to just unmorph?

Reply:
Doesn't make the trick any easier, since you still have to frame perfect jump off the crumble block to get the ledge grab.  But it does open another option for the logic since it removes needing Hi Jump, so I'll throw it in
Not really sure if it _deserves_ the higher difficulty status but 🤷